core/src/test/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -199,26 +49,13 @@ public void testIsTimeout() throws IOException {
     }
 
     public void testClusterHealth() throws IOException {
-        ShardCounter counter = new ShardCounter();
```

Comment:
How would you catch if someone delegated to the wrong method? if the API was `ClusterHealthResponse.getHealth().METHOD_NAME` I would have agreed, but we chose to keep it the same as before and have `ClusterHealthResponse.METHOD_NAME` so we need to test we delegate to the right method.